server/routes/induction/common/hopingToWorkOnReleaseController.ts

```diff
@@ -39,6 +39,6 @@ export default abstract class HopingToWorkOnReleaseController extends InductionC
 
 const toHopingToWorkOnReleaseForm = (inductionDto: InductionDto): HopingToWorkOnReleaseForm => {
   return {
-    hopingToGetWork: inductionDto.workOnRelease?.hopingToWork,
+    hopingToGetWork: inductionDto?.workOnRelease?.hopingToWork,
```

Comment:
Minor change here - in the case of creating an Induction the `InductionDto` that we get from the session is undefined.
Putting the `?` here is the right thing to do as it means the resultant form field `hopingToGetWork` is undefined which means the user will have to answer it (when we implement the POST submit handler and validation)